access/types/forms.py

```diff
@@ -33,18 +33,18 @@ def __init__(self, *args, **kwargs):
         if "exercise" not in kwargs:
             raise ConfigError("Missing exercise configuration from form arguments.")
         self.exercise = kwargs.pop("exercise")
-        self.show_correct = kwargs.pop('show_correct') if 'show_correct' in kwargs else False
-        self.show_correct_once = kwargs.pop('show_correct_once') if 'show_correct_once' in kwargs else False
+        self.model_answer = kwargs.pop('model_answer') if 'model_answer' in kwargs else False
+        self.reveal_correct = kwargs.pop('reveal_correct') if 'reveal_correct' in kwargs else False
```

Comment:
`self.reveal_correct = kwargs.pop('reveal_correct', False)` should be sufficient

Reply:
Very true, I changed those.